CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Delegated instruments are unwrapped before delegating Callbacks. (#2784)
 - Resolve supply-chain failure for the markdown-link-checker GitHub action by calling the CLI directly. (#2834)
 - Remove import of `testing` package in non-tests builds. (#2786)
+- modify the Fscanf function param in example/fib/app.go line:69 to avoid process break by "unexpected newline". (#2860)
```

Comment:
TBH I do not think it is needed (or even worth mentioning)

Reply:
Ok，I just follow the contribution guidelines.